palantir-java-format-spi/build.gradle

```diff
@@ -11,8 +11,3 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter'
     testImplementation 'com.fasterxml.jackson.core:jackson-databind'
 }
-
-tasks.withType(JavaCompile).named('compileJava') {
```

Comment:
moved all the *Compatibility blocks to the root build.gradle in the `subprojects` block. We need to set the sourceCompatibility for groovy sourceSets as well, so I think it is better to always set the source & target compatibility to 11

Reply:
I think we might avoid being able to do this by keeping `libraryTarget = 17` on the root project (so the gradle plugin, idea plugin etc use 17) then putting `libraryTarget = 17` on the `palantir-java-format` and `palantir-java-format-spi` projects

Reply:
Oops, I mean `target = 21` in the subprojects.

Reply:
changed in https://github.com/palantir/palantir-java-format/pull/1211/commits/ff5418204c120ea04c3f53023bf65f97a4f7478d